### DISCUSSION TITLE
Watchface maker naming rules

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#222] [Developer Feedback]
Hello.

There is a [guideline](https://docs.zepp.com/docs/watchface/specification/#naming-rule) related to naming rules.
Zeb exported from Watchface maker doesn't follow this guideline and files are named sequentially.
Please update the service so it would be conformant with your guidelines.
![Screenshot from 2023-07-18 16-05-51](https://github.com/zepp-health/zeppos-forum/assets/5894573/6602649e-90d0-4adc-a55a-03f16279c2bb)

**Comment by sunnyBob** [comment]
The exported names are currently incrementing numbers, which indeed makes it inconvenient for developers to locate images easily. We will optimize these names, and we greatly appreciate your suggestions.
